src/Resources/views/includes/sidebar/_profile.html.twig

```diff
@@ -1,17 +1,61 @@
-<a href="{% block logout_link %}{% endblock %}" class="whatwedo_crud-profile flex-shrink-0 flex border-t border-neutral-200 p-4">
-    <div class="flex items-center">
+<div
+    class="whatwedo_crud-profile relative flex-shrink-0 flex items-center inline-flex border-t border-neutral-200 p-4"
+    {{ stimulus_controller('araise/core-bundle/dropdown', {'alignment': 'top'}) }}
+>
+
+    <button
```

Comment:
Nit: I didn't check the dropdown controller, so please quickly verify there is either an `aria-expanded` or `aria-pressed` or similar marked added by it to ensure, the button meaning is correctly reflected in the accessibility tree.

Reply:
No, there's nothing set like that. We should integrate this in the dropdown controller absolutely

Reply:
Added to this issue: https://github.com/araise-dev/CoreBundle/issues/14